src/main/assembly/dist/bin/easy-bag-index

```diff
@@ -2,9 +2,14 @@
 
 BINPATH=`command readlink -f $0 2> /dev/null || command grealpath $0 2> /dev/null`
 APPHOME=`dirname \`dirname $BINPATH \``
+FHS_CONFIG=/etc/opt/dans.knaw.nl/easy-bag-index/
```

Comment:
the changes in this file are a left-over from configurations in the new location; copied from easy-bag-store